packages/shared/src/components/ShareBar.spec.tsx

```diff
@@ -99,14 +112,18 @@ describe('ShareBar Test Suite:', () => {
     });
   });
 
-  it('should render the Facebook button and have the correct link', async () => {
+  it('should render the Facebook button and navigate to correct link', async () => {
     renderComponent();
-    const link = await screen.findByTestId(`social-share-Facebook`);
+    const btn = await screen.findByTestId(`social-share-Facebook`);
 
-    expect(link).toHaveAttribute(
-      'href',
-      getFacebookShareLink(defaultPost.commentsPermalink),
-    );
+    btn.click();
```

Comment:
Non blocking but for future reference, prefer these events they wrap in act and just less prone to flakiness
```suggestion
    fireEvent.click(btn);
```